src/App.js

```diff
@@ -2,12 +2,54 @@ import React from 'react';
 
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRed } from './api/goods';
+import { GoodsList } from './components/GoodsList';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends React.Component {
+  state = {
+    goods: [],
+  }
+
+  handleLoad = async(callback) => {
+    await callback
+      .then((goods) => {
+        this.setState({ goods });
+      });
+  }
```

Comment:
rewrite to use only await (no need to use then here)

```suggestion
  getGoods = async(callback) => {
    await callback
      .then((goods) => {
        this.setState({ goods });
      });
  }
```